pkg/instance/router.go

```diff
@@ -23,4 +23,8 @@ func Routes(r *gin.Engine, authenticator gin.HandlerFunc, handler Handler) {
 	tokenAuthenticationRouter.PUT("/instances/:id/restart", handler.Restart)
 	tokenAuthenticationRouter.GET("/instances/:id/logs", handler.Logs)
 	tokenAuthenticationRouter.GET("/instances-name-to-id/:groupName/:instanceName", handler.NameToId)
+
+	tokenAuthenticationRouter.POST("/deployments", handler.SaveDeployment)
```

Comment:
I wonder if we should put deployments into its own handler 🤔 to clearly separate it from `instances` which it is.

Reply:
Aren't they going to be closely related anyway?